RubeesEat.UnitTests/Model/PersonTest.cs

```diff
@@ -0,0 +1,34 @@
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+[TestFixture]
+public class PersonTest
+{
+    [Test]
+    public void Person_Constructor_AssignsPropertiesCorrectly()
+    {
+        Guid expectedGuid = Guid.NewGuid();
+        string expectedFirstName = "test";
+        string expectedLastName = "test";
+        var person = new Person(expectedGuid, expectedFirstName, expectedLastName);
+        Assert.That(person.Id, Is.EqualTo(expectedGuid));
+        Assert.That(person.FirstName, Is.EqualTo(expectedFirstName));
+        Assert.That(person.LastName, Is.EqualTo(expectedLastName));
```

Comment:
Having a bunch of lines clumped together like that makes the code harder to read. We can use blank lines to separate parts of the method that do separate things. In the case of a test the most obvious case are the parts `arrange - act - assert`. Blank lines between these parts make the test a bit more readable.

In tests, we can prefer readability over repetition. As such, I would suggest inlining the `expectedFirstName` and `expectedLastName`. There is also a problem with the values you chose. As first and last name are the same value, we don't test that first and last name are assigned to the correct field. So we might have `_firstName = lastName;` in the constructor and the test still passes. Also, human readable names that fit the context are better than random test string. So using `Max Muster`, for example, is better than `test test2` :p

With these changes we arrive at something like this:

```suggestion
        Guid expectedGuid = Guid.NewGuid();
        
        var person = new Person(expectedGuid, "Max", "Muster");
        
        Assert.That(person.Id, Is.EqualTo(expectedGuid));
        Assert.That(person.FirstName, Is.EqualTo("Max"));
        Assert.That(person.LastName, Is.EqualTo("Muster"));
```

Please go through the other tests and apply the same things there to make them more readable and simplify them.